examples/cov3d_simulation.py

```diff
@@ -0,0 +1,128 @@
+"""
```

Comment:
Why is this in the current PR?

Reply:
You are right. It is better to include it in a  separate one. Just a check for the similar example for Cov2d.

Reply:
Ok. Just make sure it's in a separate commit from the other modifications.